spec/requests/admin_qualification_report_upload_spec.rb

```diff
@@ -62,6 +61,19 @@
       end
     end
 
+    context "when there's an error creating the tasks" do
+      it "displays an error and prompts the user to try again" do
```

Comment:
I think this spec will pass even without the transaction wrapping the work because the exception would get raised when the first task is being created so the rollback behaviour doesn't have coverage.

Reply:
Ahh. I had originally wanted to write a test where the second record caused the exception and we could check no records were created, but I couldn't figure out how to set that up. I'll have a think

Reply:
You know what, let's not stress this too much given how much time we have left. It's probably not worth the extra effort vs the other things we want to get done by the middle of next week.